syntaxes/hcl.tmGrammar.json

```diff
@@ -98,8 +98,13 @@
         "2": {
           "patterns": [
             {
-              "match": "[\\\"\\-\\w]+",
-              "comment": "Block label",
+              "match": "\\\"[^\\\"\\r\\n]*\\\"",
+              "comment": "Block label (String Literal)",
+              "name": "variable.other.enummember.hcl"
+            },
+            {
+              "match": "[[:alpha:]][[:alnum:]_-]*",
```

Comment:
```suggestion
              "match": "[[:alpha:]][[:alnum:]_-]+",
```

Reply:
This file is autogenerated, so we can use the comment in the YAML file